prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs

```diff
@@ -15,6 +15,7 @@ public class LicenseDetailConfiguration : SeededTable<LicenseDetail>
         public static readonly DateTime LicenceClassUpdateAug5th2022 = new(2022, 8, 5, 8, 0, 0, DateTimeKind.Utc);
         public static readonly DateTime EnableNonPrescriberPrefixAug10th2022 = new(2022, 8, 10, 8, 0, 0, DateTimeKind.Utc);
         public static readonly DateTime SetPractIdToMandatorySept6th2022 = new(2022, 9, 6, 8, 0, 0, DateTimeKind.Utc);
+        public static readonly DateTime SetPractIdToMandatoryFeb1st2023 = new(2023, 2, 1, 8, 0, 0, DateTimeKind.Utc);
```

Comment:
Can you make this Feb 2 as that date is the "special" date?

Reply:
Yes. Just to confirm, you mean to change the name to `SetPractIdToMandatoryFeb2nd2023`?

Reply:
And `new(2023, 2, 2, 8, 0, 0, DateTimeKind.Utc);` just to be clear

Reply:
You got it